package.json

```diff
@@ -20,6 +20,7 @@
     "@react-three/fiber": "^8.12.0",
     "firebase": "^10.8.0",
     "framer-motion": "^10.16.5",
+    "obscenity": "^0.2.0",
```

Comment:
Obscenity is picking up words like Duck. It might be a bit too strict. Maybe check out bad-words? they have the isProfane()